test/algorithms/ASB07.jl

```diff
@@ -0,0 +1,42 @@
+@testset "ASB07 algorithm" begin
+
+    # example 1 from
+    # "Reachability analysis of linear systems with uncertain parameters and inputs"
+    # (Adapted from Reachability.jl)
+
+    # initial set
+    X0 = BallInf([1.0, 1.0], 0.1)
+
+    # linear ODE: x' = Ax
+    A = IntervalMatrix([-1.0 ± 0.05 -4.0 ± 0.05;
+                         4.0 ± 0.05 -1.0 ± 0.05])
+    P_lin = @ivp(x' = Ax, x(0) ∈ X0)
+
+    # TODO affine ODE: x' = Ax + Bu
+    # B = IntervalMatrix(hcat([1.0 ± 0.01; 1.0 ± 0.0]))
+    # u = ConstantInput(Interval(-0.05, 0.05))
+    # P_aff = @ivp(x' = Ax + Bu, x(0) ∈ X0)
+
+    f1(P) = solve(P, tspan=(0.0, 1.0), alg=ASB07(δ=0.04));
+
+    f2(P) = solve(P, tspan=(0.0, 5.0), alg=ASB07(δ=0.04, max_order=10),
+                  order_discretization = 4);
+
+    f3(P) = solve(P, tspan=(0.0, 1.0), alg=ASB07(δ=0.04),
+                  order_discretization = 4, vars =[1, 2],
```

Comment:
same here. for the `vars` and `partition` options, those are currently ignored so we can just coment `f3` and `f4`.